modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/repository/RepositoryDistributionSnapshot.java

```diff
@@ -434,4 +444,97 @@ public DistributionSnapshot readJson(InputStream in) {
                         .stream()
                         .collect(Collectors.toMap(Plugin::getId, p -> p.getRepositorySnapshot(getDoc())));
     }
+
+    /**
+     * Returns a key/value map of properties for update.
+     *
+     * @since 11.2
+     */
+    public Map<String, String> getUpdateProperties() {
+        Map<String, String> props = new HashMap<>();
+        Stream.of(TITLE_PROPERTY_PATH, PROP_NAME, PROP_VERSION, PROP_KEY).forEach(p -> props.put(p, safeGet(p)));
+        if (StringUtils.isBlank(TITLE_PROPERTY_PATH)) {
```

Comment:
Don't you want to test the get/safeGet of this property?

Reply:
You mean adding unit tests? Why not indeed

Reply:
No, I meant that you're testing is `TITLE_PROPERTY_PATH` which shouldn't be the case as it's a constant.

Reply:
Ah thanks indeed